### PR TITLE
chore(CI): change base branch to sync enterprise

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

File: .github/workflows/git_sync.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - main-*
```
Comment (zmstone) [inline .github/workflows/git_sync.yaml:7]: main-v* ?

File: .github/workflows/git_sync.yaml [hunk 2]
```diff
@@ -22,11 +23,16 @@ jobs:
...
+            EE_REF="refs/heads/enterprise"
+          else
+            EE_REF="${GITHUB_REF}-enterprise"
+          fi
           R=$(curl --silent --show-error \
             -H "Accept: application/vnd.github.v3+json" \
             -H "Authorization: token ${{ secrets.CI_GIT_TOKEN }}" \
             -X POST \
-            -d '{"title": "Sync code into enterprise from opensource", "head": "master", "base":"enterprise"}' \
+            -d '{"title": "Sync code into enterprise from opensource", "head": "$GITHUB_REF", "base":"$EE_REF"}' \
```
Comment (zmstone) [inline .github/workflows/git_sync.yaml:35]: we may need different titles ? 
maybe: "Sync code from opensource $GITHUB_REF to entperprise $EE_REF"

File: .github/workflows/git_sync.yaml (hunk 1 above)
Comment (Rory-Z) [reply .github/workflows/git_sync.yaml]: Updated

File: .github/workflows/git_sync.yaml (hunk 2 above)
Comment (Rory-Z) [reply .github/workflows/git_sync.yaml]: Updated

**Review by zmstone — APPROVED** [review]

**Pull request merged**